class_6/exercise_11/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formulario de Bienvenida</title>
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Ingrese sus datos</h1>
+    <form id="welcomeForm">
+        <div>
+            <label for="name">Nombre:</label>
+            <br>
+            <input type="text" id="name" name="name" required> 
+            <!-- ver si va algo aca -->
+        </div>
+
+        <div>
+            <label for="age">Edad:</label>
+            <br>
+            <input type="number" id="age" name="age" required min="0"> 
+            <!-- ver si va algo aca -->
+        </div>
+
+        <div>
+            <button type="submit">Enviar</button> 
+        </div>
+
+    </form>
+
+    <div id="message"></div>
```

Comment:
Using the `div` is correct but semantically and in terms of accessibility, it's more accurate to use a `p` for text content: 

```suggestion
    <p id="message"></p>
```

Reply:
bueno, se ve me quede con el chip del proyecto que estoy trabajando y sin querer puse el comment en ingles jajajaj

Reply:
Perfecto, quedo implementado el cambio!
No hay problema por el inglés, sirve para practicar jaja